tracer/resources/args_parser.py

```diff
@@ -16,7 +16,7 @@
 # 02110-1301, USA.
 #
 
-import argparse
+import argparse, argcomplete
```

Comment:
Note that this is packaged on other distros than Fedora where this package may not be available. There is no reason for it to be required.